wrangler-storage/src/main/java/io/cdap/wrangler/store/recipe/RecipeStore.java

```diff
@@ -133,6 +139,47 @@ public Recipe getRecipe(RecipeId recipeId) throws RecipeNotFoundException {
     }, RecipeNotFoundException.class);
   }
 
+  public RecipeListResponse listRecipes(NamespaceSummary namespace, Integer pageSize, String pageToken, String sortBy) {
+    return TransactionRunners.run(transactionRunner, context -> {
+      List<Recipe> recipes = new ArrayList<>();
+      StructuredTable table = context.getTable(TABLE_ID);
+
+      // By default, sort by recipe name
+      Field<?> sortByField = Fields.stringField(RECIPE_NAME_FIELD, pageToken);
+      SortOrder sortOrder = SortOrder.ASC;
```

Comment:
Good to have this configurable. You can still continue to have default values based on sort field.

Reply:
Done